app/views/devise/registrations/edit.html.haml

```diff
@@ -1,19 +1,26 @@
 %h2.col-sm-offset-2
   Edit #{resource_name.to_s.humanize}
-= form_for resource, as: resource_name, url: registration_path(resource_name), html: { method: :put, class: 'form-horizontal' }  do |f|
+= form_for resource, as: resource_name, url: registration_path(resource_name), html: { multipart: true, method: :put, class: 'form-horizontal' }  do |f|
   = devise_error_messages!
   .form-group
+    = image_tag f.object.photo.try(:file_url, :thumb)
+  .form-group
+    = f.fields_for :photo_attributes do |p|
```

Comment:
nope, it should be :photo,  I mean, if user doesn't have photo, maybe need to make 

current_user.build_photo unless current_user.photo 

in edit action